### ISSUE TITLE
Divider text in TaskEditor panels not localized

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#10365]
[Forum post](https://forum.bryntum.com/viewtopic.php?t=31150)

Hi,

In Gantt Chart version 5.6.2, there is a localization issue for the fields shown in the attached file when using the Turkish (TR) language in the edit popup. I tried to manually adjust the TR localization file, but I wasn’t successful.

Could you please assist with this issue?

Thanks
![tr-localization-problem-2.png](https://forum.bryntum.com/download/file.php?id=31940)
![tr-localization-problem-1.png](https://forum.bryntum.com/download/file.php?id=31939)

**Comment by arcady-zherdev** [comment]
Can be fixed with `localizableProperties` help:
```js
dataset : {
    text : 'L{Constraint}'
},
localizableProperties : ['dataset.text'],
```

**Comment by arcady-zherdev** [comment]
Hi @sefadogan 

No you don't need to adjust anything. It's already fixed in the code and is going to be part of v6.1.4 release.

Best ragards,
Arcady
